spec/services/provider_interface/application_data_export_spec.rb

```diff
@@ -41,56 +41,59 @@ def expect_row_to_match_application_choice(row, application_choice)
+        'Date of birth' => application_choice.application_form.date_of_birth,
+        'Nationality code' => 'GB US',
+        'Nationality' => 'United Kingdom, United States',
+        'Disability support request' => application_choice.application_form.disability_disclosure,
+        'Email address' => application_choice.application_form.candidate.email_address,
+        'Phone number' => application_choice.application_form.phone_number,
+        'Contact address line 1' => application_choice.application_form.address_line1,
+        'Contact address line 2' => application_choice.application_form.address_line2,
+        'Contact address line 3' => application_choice.application_form.address_line3,
+        'Contact address line 4' => application_choice.application_form.address_line4,
+        'Contact postcode' => application_choice.application_form.postcode,
+        'Contact country' => 'United Kingdom',
+        'Contact country code' => 'GB',
+        'Domicile code' => application_choice.application_form.domicile,
+        'English is main language' => 'TRUE',
+        'English as a foreign language assessment details' => application_choice.application_form.english_language_details,
+        'Course' => application_choice.current_course.name,
+        'Course code' => application_choice.current_course.code,
+        'Training provider' => application_choice.current_provider.name,
+        'Training provider code' => application_choice.current_provider.code,
+        'Accredited body' => application_choice.current_accredited_provider&.name,
+        'Accredited body code' => application_choice.current_accredited_provider&.code,
+        'Location' => application_choice.current_site.name,
+        'Location code' => application_choice.current_site.code,
+        'Full time or part time' => 'Full time',
+        'Course start date' => application_choice.current_course.start_date,
+        'Has degree' => application_choice.application_form.degrees_completed ? 'TRUE' : 'FALSE',
+        'Type of degree' => first_degree&.qualification_type,
+        'Subject of degree' => first_degree&.subject,
+        'Grade of degree' => first_degree&.grade,
+        'Start year of degree' => first_degree&.start_year,
+        'Award year of degree' => first_degree&.award_year,
+        'Institution of degree' => first_degree&.institution_name,
+        'Type of international degree' => first_degree&.non_uk_qualification_type,
+        'Equivalency details for international degree' => first_degree&.composite_equivalency_details,
+        'Institution of international degree' => nil, # included for backwards compatibility. This column is always blank
+        'GCSEs' => nil,
+        'Explanation for missing GCSEs' => nil,
+        'Offered at' => application_choice.offered_at,
+        'Recruited date' => application_choice.recruited_at,
+        'Rejected date' => application_choice.rejected_at,
+        'Was automatically rejected' => 'FALSE',
```

Comment:
could we set the application choice to a specific state to avoid any flakey tests?